src/metabase/models/interface.clj

```diff
@@ -471,8 +471,8 @@
 (methodical/prefer-method! #'t2.before-insert/before-insert :hook/timestamped? :hook/entity-id)
 
 ;; --- helper fns
-(defn pre-update-changes
-  "Returns the changes used for pre-update hooks.
+(defn row-with-changes
```

Comment:
`pre-update-changes` was a bit confusing, one can think of it as the "changes" before the "update".